### DISCUSSION TITLE
Interval Server Error when I click on Your Profile button

### DISCUSSION THREAD

**Discussion by lulunac27a** [devactivity-app/devActivity-Community#29] [Report a Bug]
When I click on **Your Profile** button, the webpage gets an Interval Server Error (Error 500) and the webpage keeps reloading forever.

**Comment by olegcherev** [comment]
> When I click on **Your Profile** button, the webpage gets an Interval Server Error (Error 500) and the webpage keeps reloading forever.

checking 👀

**Reply by olegcherev** [reply]
@lulunac27a Hi, we deployed new version that should fix mentioned issue. Can you please check?

**Reply by lulunac27a** [reply]
It's working now.
